x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```
@@ -791,6 +794,90 @@
                 }
               }
             },
+            "php": {
+              "properties": {
+                "agent": {
+                  "properties": {
+                    "version": {
+                      "type": "array",
```

It is preferred to give each property a `_meta.description` e.g.
```suggestion
                      "type": "array",
                      "_meta": {
                        "description": "version of the php agent"
                      },
```